operator/controllers/csiscaleoperator_controller.go

```diff
@@ -2381,3 +2398,23 @@ func setDefaultDriverEnvValues(data map[string]string) {
 		data[config.CSIDaemonSetUpgradeMaxUnavailable] = ""
 	}
 }
+
+// listGUIPasswdExpiredClusters returns a list having clusterIds whose password is expired
+func listGUIPasswdExpiredClusters(clusters []csiv1.CSICluster) []string {
+	logger := csiLog.WithName("listPasswdExpiredClusters")
+	logger.Info("Checking each cluster if its GUI password expired")
+	expiredGui := []string{}
+	for _, cls := range clusters {
+		if conn, ok := scaleConnMap[cls.Id]; ok {
+			_, err := conn.GetClusterId(context.TODO())
+			if err != nil && isUnauthorized(err) {
+				expiredGui = append(expiredGui, cls.Id)
+			}
+		}
+	}
+	return expiredGui
+}
+
+func isUnauthorized(err error) bool {
+	return strings.Contains(err.Error(), "Unauthorized")
```

Comment:
Existing const ErrorUnauthorized="401: Unauthorized" does not work?

Reply:
Have used constant. would resolve this issue.